roles/ovirt_imageio/tasks/main.yml

```diff
@@ -85,7 +85,7 @@
   tags: daemon
   
 - name: Mount NFS volume
-  command: mount -t nfs {{ storage_nfs_export }} {{ mount_path }}
+  command: mount -t nfs -o rw,nolock,soft,intr,timeo=180,vers=3 {{ storage_nfs_export }} {{ mount_path }}
```

Comment:
nfs options is usually specified through conf file on openstack. Can we consider a similar approach here? JWTire changed default wsize and rsize. May be for later releases.